Makefile

```diff
@@ -2,12 +2,18 @@ BUILDDIR = build
 BUILD_COMMAND = sphinx-build -a -W
 BUILD_COMMAND_AND_ARGS = $(BUILD_COMMAND)
 
-docs:
+# if we are in CI we need to add python's binpath:
+export PATH := $(shell hab pkg path core/python2)/bin:$(PATH)
```

Comment:
Can you clarify how/why this is only evaluated in CI and not locally?

Reply:
This is a misleading comment - we just need to add python to the path.